app/components/questions/RentBurdenRates.js

```diff
@@ -21,34 +21,39 @@ const EMPTY_AREA = {
   rentBurdenRateFormatted: "?",
 };
 
-function transformData(response, error, selectedArea) {
+function transformData(response, error, selectedArea, selectedYear) {
   if (!response || error) {
     return {
       chartData: [],
       areaData: EMPTY_AREA,
     }
   }
+  
+  var rentYear = ((selectedYear === 2019) ? 'rent_burdened_2019'
```

Comment:
You could make the selector data have the metric name as options to avoid this check. For example:

```js
function YearSelector(props) {
  ...
  <select>
    {props.data.map((d, i) => (
      <option value={d.metric}>{d.year}</option>
    ))}
  </select>
  ...
}
...
const years = [
  { year: 2018, metric: 'rent_burdened_2018' },
  ...
]
...
<YearSelector data={years} ... />
```

Reply:
Uhm... yeah I think I'll just keep it XD
True. It is awkward regardless.